besu/src/main/java/org/hyperledger/besu/chainimport/RlpBlockImporter.java

```diff
@@ -64,7 +64,8 @@
    * @throws IOException On Failure
    */
   public <C> RlpBlockImporter.ImportResult importBlockchain(
-      final Path blocks, final BesuController<C> besuController) throws IOException {
+      final Path blocks, final BesuController<C> besuController, final boolean skipPowValidation)
```

Comment:
Do you think it's worth it to just have this take a `HeaderValidationMode` instead of the boolean? Same number of args but with a lot more flexibility.

Reply:
The boolean means different things to three different validation steps.  You will note once I skip detached rules, and another time I only do detached rules, and yet another place I do fill or none.  So it would then become three args, of which they should only be passed in two coordinated sets (`DETACHED_ONLY`/`SKIP_DETATCHED`/`FULL` and `LIGHT_DETACHED_ONLY`/`LIGHT_SKIP_DETATCHED`/`LIGHT`).  No need to bring that complexity into the function call.

Reply:
Makes sense!